custom-routes-vis/src/main/webapp/script.js

```diff
@@ -333,47 +324,53 @@ function createRouteFromCoordinates(
  * @param {num} routeNum The index of the selected route in the routes array.
  * @param {num} totalDurationSec The duration of the route in seconds.
  * @param {num} totalDistanceMeters The distance of the route in meters.
- * TODO(chenyuz): Add in the route token returned from the Routes Preferred API.
+ * @param {string} routeToken The route token of the currently selected route.
  */
-function selectRouteDisplayDetails(routeNum, totalDurationSec, totalDistanceMeters) {
+function selectRouteDisplayDetails(routeNum, totalDurationSec, totalDistanceMeters,
+    routeToken) {
   displayedRoutes[selectedRouteNum].setOptions({ strokeOpacity: 0.3, })
   selectedRouteNum = routeNum;
   displayedRoutes[routeNum].setOptions({ strokeOpacity: 1.0, });
 
   let routeInfoElement = document.getElementById('route-info');
   routeInfoElement.innerText = 'Selected Route Info:' +
       '\nDuration: ' + formatDuration(totalDurationSec) +
-      '\nDistance: ' + formatDistance(totalDistanceMeters) +
-      '\nRoute Token: ';
-}
+      '\nDistance: ' + formatDistance(totalDistanceMeters);
 
-/** Shows or hides rate-card-data div when service endpoint changed. */
-function changeServiceEndpoint() {
-  let serviceEndpoint = document.getElementById('service-endpoint').value;
-
-  // Only display rate card div if compute custom routes is selected.
-  document.getElementById('rate-card-data').style.display = 
-      (serviceEndpoint === 'compute-routes-alpha') ? 'block' : 'none';
+  if (routeToken !== '') {
+    routeInfoElement.innerText += '\nRoute Token: check the developer console';
+    console.log(routeToken);
```

Comment:
Do we want some sort of a message before printing out the token?
i.e. something to indicate that this is the route token

Reply:
+1

Reply:
ok